roles/openshift_node40/tasks/create_files_from_ignition.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Create all the directories we will need
+  command: "mkdir -p {{ item }}"
+  with_items: "{{ l_parse_ignition_dict.dir_list }}"
+
+- name: create files from ignition contents
```

Comment:
nit - for consistency with the above task, you can capitalize the first letter `Create`